device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/FeedbackStatusCodeEnum.java

```diff
@@ -1,13 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-package com.microsoft.azure.sdk.iot.device;
-
-public enum FeedbackStatusCodeEnum
-{
-    none,
```

Comment:
No uses anywhere, not sure what this existed for

Reply:
the enum values look similar to what c2d message feedback uses on a service client, so this might be a duplicate/redundant class, right?

Reply:
That was probably the idea. But for whatever reason it wasn't used in the device client so we can remove it